src/content/docs/releases/index.mdx

```diff
@@ -28,10 +26,6 @@ Every release includes changes to one or more codebases described in the [compat
 > MINOR version when you add functionality in a backward-compatible manner  
 > PATCH version when you make backward-compatible bug fixes  
 
-<Aside type="note">
-The "Storefront suite" version is used for marketing purposes and does not conform to semantic versioning.
-</Aside>
-
```

Comment:
We are no longer using version numbers for the storefront suite version so I removed this section.

Instead of `1.0.0` we're naming them by release date (ex: "December 2024")